### ISSUE TITLE
about "#define load_game (*((void(*)(const char* filename))(0x7ffc/2)))"

### ISSUE THREAD

**Issue by sunkechenjing** [Rodot/Gamebuino#29]
#define load_game (*((void(*)(const char* filename))(0x7ffc/2))) in gamebino.h  
I think this command to game load task,but i can't this commad is how to finish the work.I think this command is related to gamebuino bootloader.

**Comment by Sorunome** [comment]
That is correct, it is just a macro to call an address within the bootloader as if it was a normal function. Said bootloader function then re-flashes the gamebuino with the .HEX file specified

Am 13. März 2018 09:52:04 MEZ schrieb sunkechenjing <notifications@github.com>:
